frontend/src/components/SidebarContainer/sidebarContainer.styled.js

```diff
@@ -4,8 +4,8 @@ export const Container = styled.div`
   background-color: ${(props) => props.theme.main};
   padding: 20px 15px;
   min-height: 100vh;
-  width: 300px;
-  flex: 1 0 300px;
+   width: 350px;
   position: relative;
+  transition: all 2s linear;
```

Comment:
This transition is just transitioning the background color of the sidebar when I toggle the light/dark mode.